sdk/storage/storage-blob/src/generated/src/models/index.ts

```diff
@@ -4777,11 +4777,6 @@ export interface BlobCreateSnapshotHeaders {
    * provided in the request and encrypted with a customer-provided key.
    */
   isServerEncrypted?: boolean;
-  /**
```

Comment:
@XiaoningLiu & @jiacfan was a different swagger json file used last time Or this field really got removed?

Reply:
@jeremymeng This was removed by Sean McCullough on 8/29/2019. Refer [here](https://msazure.visualstudio.com/One/_git/Storage-XStore/commit/9373039d686076be8507f9c7060a0eabff89da8b?path=%2Fsrc%2FXFE%2FOpenApi%2FMicrosoft.BlobStorage%2Fstable%2F2019-02-02%2Fblob.json&gridItemType=2&mpath=%2Fsrc%2FXFE%2FOpenApi%2FMicrosoft.BlobStorage%2Fstable%2F2019-02-02%2Fblob.json&opath=%2Fsrc%2FXFE%2FOpenApi%2FMicrosoft.BlobStorage%2Fstable%2F2019-02-02%2Fblob.json&mversion=GC9373039d686076be8507f9c7060a0eabff89da8b&oversion=GC8d98ce77a4490a6500b8e1806f1d6124d201ae1b&_a=compare)